README.md

```diff
@@ -11,6 +11,19 @@ This is an centos7 based image for running [fluentd](http://fluentd.org). It is
 
 This work is based on the [docker-fluentd](https://github.com/fabric8io/docker-fluentd) and [docker-fluentd-kubernetes](https://github.com/fabric8io/docker-fluentd-kubernetes) images by the fabric8 team. This image is in with [deis](https://github.com/deis/deis) v2 to send all log data to the [logger](https://github.com/deis/logger) component.
 
+### Enable more verbose logging
+By default we do not capture kubernetes system logs. However, it is possible to tell fluentd to capture those logs just by specifying a few new environment variables.
+
+* CAPTURE_START_SCRIPT
+* CAPTURE_DOCKER_LOG
+* CAPTURE_ETCD_LOG
+* CAPTURE_KUBELET_LOG
+* CAPTURE_KUBE_API_LOG
+* CAPTURE_CONTROLLER_LOG
+* CAPTURE_SCHEDULER_LOG
+
+Our logic only checks that the value has a length greater than `1` so you can set the value to anything. You should make these changes to your `deis-logger-fluentd-daemon.yaml` file in the `tpl` directory of the Workflow chart. 
```

Comment:
Really `test -n` checks that it isn't an empty string, so maybe a better description would be:
> Set a variable's value to a non-empty string such as "true" to capture that log. Make these changes to the `tpl/deis-logger-fluentd-daemon.yaml` file in the Workflow chart directory.